doc/main.dox

```diff
@@ -172,13 +172,27 @@ is to use a camera-specific variant of a subtask. Examples include:
 - lsst.obs.sdss.selectSdssImages.SelectSdssImagesTask: an version of the task that selects images
     for co-addition of SDSS stripe 82 images
 
-Here is an example of retargeting a subtask in a config override file; this retargets
-lsst.pipe.tasks.exampleCmdLineTask.ExampleCmdLineTask `stats` with a simpler version:
+How you retarget a subtask depends if the subtask is specified as an
```

Comment:
I will chnage "if" to whether".

I believe "an" is correct before "LSST" since the L is pronounced "ell".